src/analysis/calculus/affine_map.lean

```diff
@@ -33,4 +33,32 @@ begin
   exact times_cont_diff_const,
 end
 
+/-- The evaluation map on the space of continuous affine maps is smooth. -/
+lemma times_cont_diff_apply {n : with_top ℕ} :
```

Comment:
@fpvandoorn Does your [library note](https://leanprover-community.github.io/mathlib_docs/notes.html#continuity%20lemma%20statement) apply here?

Reply:
It certainly does! You want something like `times_cont_diff f -> times_cont_diff g -> times_cont_diff (\la x, (f x) (g x)` where `f : X -> (V →A[𝕜] W)`.

Reply:
Thanks, now done.